core/src/App.html

```diff
@@ -622,8 +622,10 @@
   //// SPLIT VIEW
   export const openSplitView = (nodepath, settings) => {
     if (mfSplitView.displayed) {
-      console.warn('Only one splitview can be oppened at a time');
+      console.warn('Only one splitview can be opened at a time');
       return;
+    } else {
+      mfSplitView = SplitViewSvc.getDefaultData().mfSplitView;
     }
     SplitViewSvc.open(getComponentWrapper(), nodepath, settings);
```

Comment:
```suggestion
    }
    mfSplitView = SplitViewSvc.getDefaultData().mfSplitView;
    SplitViewSvc.open(getComponentWrapper(), nodepath, settings);
```
Does this make sense and work properly?

Reply:
jap, good point :)